engine/userguide/storagedriver/overlayfs-driver.md

```diff
@@ -27,7 +28,7 @@ OverlayFS is supported if you meet the following prerequisites:
   
 - The `overlay` driver is allowed but not recommended for Docker CE.
 
-- Version 4.0 or higher of the Linux kernel. If you use an older kernel, you
+- Version 4.0 or higher of the Linux kernel, or RHEL/Centos 3.10.0-693 or higher. If you use an older kernel, you
```

Comment:
Is that a kernel version or a RHEL / Centos version? I'm guessing the former, so maybe clarify. I think the acronym is RHCK (Red Hat compatible kernel).

Reply:
3.10.0-693 is the correct kernel for Rhel/Centos 7.4.